src/core_plugins/kibana/public/dashboard/reducers/index.js

```diff
@@ -0,0 +1,42 @@
+import { combineReducers } from 'redux';
+import {
+  embeddables,
+  getEmbeddableTitle as getEmbeddableTitleFromEmbeddables,
+  getEmbeddableEditUrl as getEmbeddableEditUrlFromEmbeddables,
+  getEmbeddableError as getEmbeddableErrorFromEmbeddables,
+  getEmbeddable as getEmbeddableFromEmbeddables,
+} from './embeddables';
+
+import {
+  panels,
+  getPanel as getPanelFromPanels,
+  getPanelType as getPanelTypeFromPanels
+} from './panels';
+
+import {
+  view,
+  getViewMode as getViewModeFromView,
+  getFullScreenMode as getFullScreenModeFromView,
+  getMaximizedPanelId as getMaximizedPanelIdFromView
+} from './view';
+
+export const dashboard = combineReducers({
+  view,
+  panels,
+  embeddables,
+});
+
+export const getPanels = state => state.panels;
```

Comment:
Management is doing this too, putting selectors in reducer files. Is that the plan going forward? Any reason you don't separate the two?

Reply:
I actually think we should split them out because I ran into a circular reference error, but didn't want to make that decision in this PR.

Reply:
Alright. We split it out in Canvas, but I think Cloud might not. It would probably be good to get consensus on the organization, but not something we need in this PR.